lib/worker.js

```diff
@@ -25,6 +25,7 @@ const { validateParameters } = require('./validate');
 const Timer = require('./utils/timer');
 const { Sampler } = require('@adobe/metrics-sampler');
 const { metrics: cgroupMetrics, cpu } = require('@adobe/cgroup-metrics');
+const { fail } = require('assert');
```

Comment:
was this imported on purpose?

Reply:
Not sure, it was previously required but not used anywhere and eslint was complaining.

Reply:
I would remove it, should not be necessary in this file